design/service-binding.md

```diff
@@ -202,9 +202,33 @@ spec:
         displayName: Bootstrap servers
         path: status.bootstrap
         x-descriptors:
-        - 'urn:alm:descriptor:servicebinding:secret:endpoints'
+        - binding:env:object:secret:endpoints
```

Comment:
Am I correct the `env` in `binding:env:object` indicates the intent to bind this property as an environment variable? As mentioned above, it might make more sense to let the ServiceBinding CR itself decide whether to volume mount the binding secret or import it as environment variables.

Reply:
Agreed.